packages/openapi3/src/lib.ts

```diff
@@ -40,12 +40,6 @@ const libDef = {
         default: "Duplicate @statusCode declarations on response type",
       },
     },
-    "duplicate-body-types": {
-      severity: "error",
-      messages: {
-        default: "Request has multiple body types",
-      },
-    },
```

Comment:
This diagnostic was never used, so I removed it.  There is a `duplicate-body` diagnostic that seems to cover any case where this might be used.

Reply:
must be a copy and paste from the autorest library